docs/howtos/policy-servers/01-custom-cas.md

```diff
@@ -4,7 +4,7 @@ title: Using custom certificate authorities
 description: Using custom certificate authorities with Kubewarden policy servers.
 keywords: [kubewarden, kubernetes, custom certificate authorities]
 doc-persona: [kubewarden-operator, kubewarden-integrator]
-doc-type: [howto, reference]
+doc-type: [howto]
```

Comment:
Both this page `docs/howtos/policy-servers/01-custom-cas.md` and `howtos/custom-certificate-authorities.md` explain the `sources.yaml` config file for the policy-server. I would leave an example of usage and merge both `sources.yaml` config file explanations under a reference page.

Reply:
OK. I will need some more input. I expect I'm not understanding it but if they are referencing the same format `sources.yaml` why do the keys look different? `insecure_sources` vs `insecureSources`. Thanks

Reply:
You are right, they are indeed 2 different things.

There are:
1. The "`sources.yaml` file format", which is used by both `kwctl` and the `policy-server` binaries. This `sources.yaml` file format should go into the references page. It is the one at "distributing-policies/custom-certificate-authorities".
2. The PolicyServer `spec.sourceAuthorities` and `spec.insecureSources`. They are part of the the CRD definition, but too big to be part of the CRD reference page. IMHO  `spec.sourceAuthorities` and `spec.insecureSources` both should be part of a new reference page. The CRDs should be updated to include links from spec.sourceAuthorities and spec.insecureSources into that new reference page for them. They are closely related to the "`sources.yaml` file format", in that the controller reads the PolicyServer spec.sourceAuthorities and spec.insecureSources and creates a sources.yaml for it, when instantiating its deployment.

Reply:
Hmm, is it acceptable to create a new card to do this.? Independent of the rest of the restructuring. To be done next. Not left languishing. :smiley:

Reply:
Indeed, sorry! Let's move this forward and iterate. Approving.

Reply:
Ok. New issue in https://github.com/kubewarden/docs/issues/354